data/json/mapgen/shooting_range.json

```diff
@@ -133,7 +133,7 @@
       },
       "place_signs": [
         { "signage": "<family_name> Family Shooting Range", "x": 10, "y": 4 },
-        { "signage": "Open sunrise to sunset, everday besides Sundays", "x": 13, "y": 4 },
+        { "signage": "Open sunrise to sunset, everyday besides Sundays", "x": 13, "y": 4 },
```

Comment:
you want "every day" two words here

Reply:
I prefer it the way it is, the whole thing has a bit of ungrammatical flavor to it, I just got rid of the obvious misspelling.